hooks/tk-multi-loader2/tk-katana_actions.py

```diff
@@ -136,26 +136,34 @@ def execute_action(self, name, params, sg_publish_data):
         # resolve path
         path = self.get_publish_path(sg_publish_data)
 
-        if name == "open_project":
-            self._open_project(path, sg_publish_data)
+        if name == "import_project":
+            self._import_project(path, sg_publish_data)
```

Comment:
Since `sg_publish_data` is unused, we can probably not call using it

```suggestion
            self._import_project(path)
```